test/test_ops.py

```diff
@@ -422,12 +458,15 @@ def test_boxes_shape(self):
 
 
 class TestMultiScaleRoIAlign:
+    def make_obj(fmap_names=["0"], output_size=(7, 7), sampling_ratio=2, **kwargs):
+        return ops.poolers.MultiScaleRoIAlign(fmap_names, output_size, sampling_ratio)
+
     def test_msroialign_repr(self):
         fmap_names = ["0"]
         output_size = (7, 7)
         sampling_ratio = 2
         # Pass mock feature map names
-        t = ops.poolers.MultiScaleRoIAlign(fmap_names, output_size, sampling_ratio)
+        t = TestMultiScaleRoIAlign.make_obj(fmap_names, output_size, sampling_ratio)
```

Comment:
Was this supposed to be as follows?
```suggestion
        t = self.make_obj(fmap_names, output_size, sampling_ratio)
```